web/html/src/manager/audit/cveaudit/cveaudit.tsx

```diff
@@ -219,6 +219,21 @@ class CVEAudit extends React.Component<Props, State> {
     });
   };
 
+  getPatchStatusAccuracyWarning = (row) => {
+    const dataSources: string[] = row.scanDataSources;
+    if (!dataSources) {
+      return "Unknown patch status";
+    }
+
+    if (dataSources.indexOf("OVAL") === -1) {
+      return "OVAL data not in sync. Possible false negatives";
+    } else if (dataSources.indexOf("CHANNELS") === -1) {
+      return "Channels not in sync for the given server product. Patches are not available.";
+    }
+
+    return "If you see this report a bug.";
```

Comment:
This isn't a very good way to indicate an error, I think it would make more sense to show an error toast, log an error with more details about what went wrong, and then render the field empty.

Reply:
If reasonably possible, it might be good to aggregate showing the toast though, so if the underlying data is broken and the user has 25 rows showing, they don't get 25 error toasts.

Reply:
Both solutions sound good, but I'm leaning toward the logging solution. The error in question is unlikely to occur; if it does, it's likely due to an incompatibility between the frontend and backend server, so it's better to aim for the simpler solution.

Reply:
Sounds good to me. 👍